content/posts/annoucing-slight-v0_3.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "Annoucing Slight v0.3"
+description: "Announcing new features and updates on Slight v0.3 release."
+date: 2023-02-25 00:00:00 +0000 UTC
+authorname: "Jiaxiao (Joe) Zhou"
+author: "@jiaxiao_zhou"
+authorlink: "https://twitter.com/jiaiao_zhou"
+tags: ["slight", "spiderlightning", "Wasm", "WASI"]
+---
+
+We are excited to announce the release of `Slight` v0.3, the latest version of our WebAssembly runtime that utilizes SpiderLightning (also known as `wasi-cloud`) capabilities.
```

Comment:
I would recommend not putting the name Slight in a command block in documentation or blog posts because it makes the rendered paragraph look a bit cluttered and you aren't actually giving someone a command to copy/paste (which is when the command block formatting is helpful).

Reply:
Sounds good!